components/user/UserMilestone.js

```diff
@@ -7,15 +7,29 @@ import Link from "@components/Link";
 import Edit from "@components/account/manage/Edit";
 import Markdown from "@components/Markdown";
 
+function formatDate(day, month, year, dateFormat) {
```

Comment:
We have a date format function in `services/utils` already, maybe this is better in there also, so it can be reused

Example `services/utils/dateFormat.js`

Reply:
The name is too genetic, maybe something like `shortenDate` as we are making it shorter?

Reply:
I have moved the formatDate function to ```services/utils/dateFormat.js``` and renamed it to ```shortenDate```